examples/server/main.go

```diff
@@ -98,7 +112,37 @@ func newAWSPubSubServer(logger *logrus.Logger) (pubsubgrpc.PubSubServer, error)
 		return pubsubaws.NewPublisher(sess, topic)
 	}
 	subFactory := func(ctx context.Context, topic, subscriptionID string) (pubsub.Subscriber, error) {
-		return pubsubaws.NewSubscriber(sess, topic, subscriptionID)
+		return pubsubaws.NewSubscriber(sess, sess, topic, subscriptionID)
+	}
+	return pubsubgrpc.NewPubSubServer(pubFactory, subFactory), nil
+}
+
+// newLocalStackPubSubServer creates a new grpc PubSub server using the broker
+// implementation for LocalStack
+func newLocalStackPubSubServer(logger *logrus.Logger) (pubsubgrpc.PubSubServer, error) {
```

Comment:
I feel we dont need a separate method for localstack, instead we could create two sessions for sns and sqs by reading endpoints from conf. and change the signature of `VerifyPermissions`